.vitepress/config.mts

```diff
@@ -54,7 +54,8 @@ export default defineConfig({
             items: [
               { text: "Creating your own server instance", link: "/guides/getting-started/creating-your-own-server-instance" },
               { text: "Implementing token validation in your APIs", link: "/guides/getting-started/implementing-token-validation-in-your-apis" },
-              { text: "Integrating with a remote server instance", link: "/guides/getting-started/integrating-with-a-remote-server-instance" }
+              { text: "Integrating with a remote server instance", link: "/guides/getting-started/integrating-with-a-remote-server-instance" },
+              { text: "Using dev tunnels", link: "/guides/getting-started/using-dev-tunnels" }
```

Comment:
`dev tunnels` is a bit generic, let's rename all the references to `Microsoft dev tunnels` instead.